protocol.md

```diff
@@ -1,147 +1,30 @@
-      &nbsp;&nbsp;"age": 25<br>
-      }<br>
-      ```
-      </code></td>
-    </tr>
-    <tr>
-      <td><a href="/extended-syntax/#footnotes">Footnote</a></td>
-      <td><code>
-        Here's a sentence with a footnote. [^1]<br><br>
+By providing a protocol instead of a service, we empower the users financially whilst increasing their autonomy by removing the need to rely on an over-arching bounty platform that takes money from the work of users. Sharing a completely open and deterministic payout protocol removes bias from the hiring process. We believe the early-adopters on the employer side of the protocol can gain knowledge and experience that may give them a tactical advantage in the domain of requirement specification for AI engines like GPT-3. It can be difficult to identify which development processes in a company are suitable to offload into the TruCol protocol. It can be difficult to write air-tight test specifications in sponsor contracts. We intend to work as a consultancy to assist companies in successfully navigating these challenges to help them get the most out of the TruCol protocol.
 
-        [^1]: This is the footnote.
-      </code></td>
-    </tr>
-    <tr>
-      <td><a href="/extended-syntax/#heading-ids">Heading ID</a></td>
-      <td><code>### My Great Heading {#custom-id}</code></td>
-    </tr>
-    <tr>
-      <td><a href="/extended-syntax/#definition-lists">Definition List</a></td>
-      <td><code>
-        term<br>
-        : definition
-      </code></td>
-    </tr>
-    <tr>
-      <td><a href="/extended-syntax/#strikethrough">Strikethrough</a></td>
-      <td><code>~~The world is flat.~~</code></td>
-    </tr>
-    <tr>
-      <td><a href="/extended-syntax/#task-lists">Task List</a></td>
-      <td><code>
-        - [x] Write the press release<br>
-        - [ ] Update the website<br>
-        - [ ] Contact the media
-      </code></td>
-    </tr>
-  </tbody>
-</table>
+## Implementation
 
-## Downloads
+The protocol can be implemented on smart contracts of various chains, such as Ethereum. The employer can determine their desired level of decentralization by choosing whether they want to publish the test specification completely decentralized on-chain, or make use of GitHub, continuous integration and oracles to verify the completion of the task.
 
-You can <a href="/assets/markdown-cheat-sheet.md" download="markdown-cheat-sheet.md">download this cheat sheet as a Markdown file</a> for use in your Markdown application.
+## TruCol Enthousiasts
+Are you intruiged by the TruCol protocol and would you like to read more. Do not worry, we got you. The [white paper](#) is online and contains all information you need. Or are you done with reading and would you like to tinker around with what TruCol has to offer, for information on getting started see [the getting started section](/getting-started) or check out the [TruCol github repository](https://github.com/TruCol/TruCol)
```

Comment:
Enthusiasts*, intrigued*

Reply:
Are you intrigued by the TruCol protocol, and would you like to read more? Feel free to have a look at the [white paper](#) and check out how to get started [here](/getting-started). Or directly dive into the code at the [TruCol GitHub repository](https://github.com/TruCol/TruCol).